tasties_app/forms.py

```diff
@@ -16,3 +18,23 @@ class Meta:
             'username': forms.fields.TextInput(attrs={'class': 'form-control', 'placeholder': 'Username'}),
             'email': forms.fields.EmailInput(attrs={'class': 'form-control', 'placeholder': 'john@doe.com'}),
         }
+
+
+class CreateRecipeForm(ModelForm):
+    title = forms.CharField(max_length=100, widget=forms.TextInput(attrs={'class': 'form-control'}))
```

Comment:
I'm not sure if the class should be part of the form

Reply:
This is a Model Form, which is a special type of Django form that is bound to a specific model. See [here](https://docs.djangoproject.com/en/4.1/topics/forms/modelforms/#:~:text=form%20class.%0A%3E%3E%3E-,class%20ArticleForm(ModelForm),-%3A%0A...%20%20%20%20%20class).

Reply:
I meant to this `attrs={'class': 'form-control'}`

Reply:
@GilShechter Added this. I think it should definitely be here, since it applies an html class to this input. The form will look very ugly if we couldn't style it...

Reply:
@GilShechter any thoughts on this?

Reply:
Thanks